lib/reporters/html/view-model.js

```diff
@@ -91,7 +98,11 @@ module.exports = inherit({
      * @returns {ViewModelResult}
      */
     getResult: function() {
+        this._skips = _.isEmpty(this._skips)
+            ? [{suite: 'There are no skipped tests'}]
+            : _(this._skips).collect(JSON.stringify).uniq().map(JSON.parse).value();
```

Comment:
As we have discussed we skip entire suite with all states, not a single state. As a result the suite tree contains all skipped states with the same comment. We are not interested in information like this:
```
suiteXXX > state1 > browser, reason: comment1
suiteXXX > state2 > browser, reason: comment1
...
suiteXXX > stateN > browser, reason: comment1
```
It is more convenient to see this:
```
suiteXXX > browser, reason: comment1
```
Just an example how it looks like now:
![example](https://cloud.githubusercontent.com/assets/1412876/11834350/bb0f47a4-a3dc-11e5-8c12-bf70b1a325b5.png)